cli/cmd/check.go

```diff
@@ -49,10 +52,37 @@ problems were found.`,
 			return statusCheckResultWasError(os.Stdout)
 		}
 
-		return checkStatus(os.Stdout, kubectl, kubeApi, healthcheck.NewGrpcStatusChecker(public.ConduitApiSubsystemName, conduitApi))
+		err = checkStatus(os.Stdout, kubectl, kubeApi, healthcheck.NewGrpcStatusChecker(public.ConduitApiSubsystemName, conduitApi))
+		if err != nil {
+			err = prettyPrintSystemInformation(conduitApi, kubectl, sh)
```

Comment:
Should we add a command line flag that forces this sysinfo to be print even if the test doesnt fail? Could be useful to help users debug their setup.